jellyfin_kodi/library.py

```diff
@@ -605,39 +605,37 @@ def run(self):
                 except Queue.Empty:
                     break
 
-                # Verify that the updated item is in our local whitelist
-                library = find_library(self.server, item)
-                if library:
-                    default_args = (self.server, jellyfindb, kodidb, self.direct_path, library)
-                    try:
-                        if item['Type'] == 'Movie':
-                            Movies(*default_args).movie(item)
-                        elif item['Type'] == 'BoxSet':
-                            Movies(*default_args).boxset(item)
-                        elif item['Type'] == 'Series':
-                            TVShows(*default_args).tvshow(item)
-                        elif item['Type'] == 'Season':
-                            TVShows(*default_args).season(item)
-                        elif item['Type'] == 'Episode':
-                            TVShows(*default_args).episode(item)
-                        elif item['Type'] == 'MusicVideo':
-                            MusicVideos(*default_args).musicvideo(item)
-                        elif item['Type'] == 'MusicAlbum':
-                            Music(*default_args).album(item)
-                        elif item['Type'] == 'MusicArtist':
-                            Music(*default_args).artist(item)
-                        elif item['Type'] == 'AlbumArtist':
-                            Music(*default_args).albumartist(item)
-                        elif item['Type'] == 'Audio':
-                            Music(*default_args).song(item)
-
-                        if self.notify:
-                            self.notify_output.put((item['Type'], api.API(item).get_naming()))
-                    except LibraryException as error:
-                        if error.status == 'StopCalled':
-                            break
-                    except Exception as error:
-                        LOG.exception(error)
+                default_args = (self.server, jellyfindb, kodidb, self.direct_path)
+                try:
+                    LOG.info('{} - {}'.format(item['Type'], item['Name']))
```

Comment:
Should this really be info? Looks like it would be rather noisy.
```suggestion
                    LOG.debug('{} - {}'.format(item['Type'], item['Name']))
```